lib/ui/profile/components/profile_view.dart

```diff
@@ -107,6 +105,135 @@ class ProfileView extends StatelessWidget {
                             onTap: () {},
                             onChanged: (value) {},
                           ),
+                        ],
+                        if (kDebugProfileService) ...[
```

Comment:
can we move this to a method or a widget and explain whats going on here?

Reply:
We can delete all this debug code once we have a profile edit screen.